gui/wxpython/nviz/wxnviz.py

```diff
@@ -508,14 +510,14 @@ def InitView(self):
 
         Debug.msg(1, "Nviz::InitView()")
 
-    def SetBgColor(self, color_str):
+    def SetBgColor(self, color_str: str) -> None:
         """Set background color
 
         :param str color_str: color string
         """
         Nviz_set_bgcolor(self.data, Nviz_color_from_str(color_str))
 
-    def SetLight(self, x, y, z, color, bright, ambient, w=0, lid=1):
+    def SetLight(self, x, y, z, color, bright, ambient, w=0, lid=1) -> None:
```

Comment:
Here parameters are not annotated...

Reply:
There wasn't enough info yet on the calling site side, but I can try to take a second look from the C-api side

Reply:
```suggestion
        def SetLight(
        self,
        x: float,
        y: float,
        z: float,
        color,
        bright: float,
        ambient: float,
        w: float = 0,
        lid: int = 1,
    ) -> None:
```
For color, it isn't clear if its ok, as the other parts didn't have that pattern, and the way it is used, but saying it is a string is weird.